packages/functional-tests/tests/subscription-tests/coupons-ui-tests/ui-tests.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from '../../../lib/fixtures/standard';
+
+test.describe('ui functionality', () => {
+  test.beforeEach(({ }, { project }) => {
+    test.skip(project.name !== 'stage', "Only run these tests in 'stage' env");
+  });
+
+  test('verify discount textbox display', async ({
+    pages: { relier, subscribe },
+  }) => {
+    test.slow();
```

Comment:
Same Q here about seeing if we can move the `test.slow()` into the `test.beforeEach()` hook and removing from the tests.